test/controllers/YourBillControllerSpec.scala

```diff
@@ -241,6 +241,42 @@ class YourBillControllerSpec extends ItSpec {
       }
     }
 
+    "return your bill page for SIMP" in {
+      stubCommonActions()
+      EssttpBackend.EligibilityCheck.findJourney(testCrypto, Origins.Simp.Pta)()
+
+      val result: Future[Result] = controller.yourBill(fakeRequest)
+      val pageContent: String    = contentAsString(result)
+      val doc: Document          = Jsoup.parse(pageContent)
+
+      RequestAssertions.assertGetRequestOk(result)
+      ContentAssertions.commonPageChecks(
+        doc,
+        expectedH1 = "Your Simple Assessment tax bill is £3,000",
+        shouldBackLinkBePresent = true,
+        expectedSubmitUrl = Some(routes.YourBillController.yourBillSubmit.url),
+        regimeBeingTested = Some(TaxRegime.Simp)
+      )
+
+      val tableRows = doc.select(".govuk-summary-list > .govuk-summary-list__row").asScala.toList
+      tableRows.size shouldBe 2
+
+      tableRows(0)
+        .select(".govuk-summary-list__key")
+        .text() shouldBe "13 Jul 2020 to 14 Jul 2020 Bill due 7 February 2017"
+      tableRows(0).select(".govuk-summary-list__value").text() shouldBe "£2,000 (includes interest added to date)"
+
+      tableRows(1).select(".govuk-summary-list__key").text() shouldBe "13 Aug 2020 to 14 Aug 2020 Bill due 7 March 2017"
+      tableRows(1).select(".govuk-summary-list__value").text() shouldBe "£1,000 (includes interest added to date)"
+
+      val extraText = doc.select(".govuk-body").asScala.toSeq
```

Comment:
is there anything you could stick into the tests for the other regimes that will fail if the new text accidentally shows up for them?